test/integration/http/assert_http_response.py

```diff
@@ -0,0 +1,40 @@
+from http import HTTPStatus
+from http.client import HTTPResponse
+from typing import Optional
+
+
+def assert_status_line(response: HTTPResponse, expected_code: HTTPStatus) -> None:
+    assert (
+        response.status == expected_code
+    ), f"Expected status code {expected_code}, but got {response.status}"
+    assert (
+        response.reason == expected_code.phrase
+    ), f"Expected status code reason {expected_code.phrase}, but got {response.reason}"
+    assert response.version == 11, f"Expected version 1.1, but got {response.version}"
+
+
+def assert_header(
+    response: HTTPResponse,
+    expected_header_name: str,
+    expected_header_value: str,
+) -> None:
+    response_header_value = response.getheader(expected_header_name)
+    # Header nameが存在するかどうか
+    if response_header_value is None:
+        raise ValueError(f"Header {expected_header_name} not found in the response")
+    # Header valueは末尾に空白があるためstrip()使用
+    response_header_value = response_header_value.rstrip()
+    assert (
+        response_header_value == expected_header_value
+    ), f"Expected header value {expected_header_value}, but got {response_header_value}"
+
+
+def assert_body(response: HTTPResponse, path: str) -> None:
+    response_body = response.read()
+    expected_body = b""
+    with open(path) as f:
+        expected_body = f.read().encode('utf-8')
```

Comment:
私の devcontainer で `make check` したら全ファイルの中でこの行だけこう (`"utf-8"`) 変更されたんですが、もしかしてそっちの手元では `'utf-8'` のままだったりしますか？
```suggestion 
        expected_body = f.read().encode("utf-8")
```

Reply:
あ…すみません、devcontainer 内でコマンド打ててなかっただけでした
忘れてください…

devcontainer : `""` で書いても `''` で書いても特に修正されない
vm : `''` で書くと `""` に修正される
みたいなことが起きてました